### ISSUE TITLE
Capacitor_THT: C_Rect_* have wrong descriptions

### ISSUE THREAD

**Issue by antoniovazquezblanco** [KiCad/kicad-footprints#1131]
Example:
https://github.com/KiCad/kicad-footprints/blob/ccb39e2c536404e47239b91ad5bd3abc4e689980/Capacitor_THT.pretty/C_Rect_L7.5mm_W6.5mm_P5.00mm.kicad_mod#L2

Radial should not be in the description.

I think those are scripted. I believe it is a good idea to also include that in the description.

**Comment by evanshultz** [comment]
Yes, they are scripted. See https://github.com/pointhi/kicad-footprint-generator/blob/master/scripts/tools/footprint_scripts_resistorlike.py and https://github.com/pointhi/kicad-footprint-generator/blob/master/scripts/Capacitors_THT/make_Capacitors_THT.py.

I made a few changes to the script at https://github.com/pointhi/kicad-footprint-generator/pull/200, but it didn't affect these footprints. It was clear the script isn't very flexible so more clauses could be added to write out the descriptions better for various part types.

Perhaps related, I submitted the footprints at https://github.com/KiCad/kicad-footprints/pull/1019 and noted that KLC specifies a different format (including height) which are current library does not capture (even though height is known and fixed). Perhaps some renaming was in order anyway??
